.github/workflows/test-spec.yml

```diff
@@ -34,10 +34,8 @@ jobs:
             tests: snapshot
           - version: 2.17.0
             hub: opensearchstaging
-            ref: '@sha256:6398c27d7560626ed6b0ba28b3d6b20b7f00c6d94abf45ad3a820f8eeb3d61a3'
```

Comment:
removed fixed SHA's as they were causing CI failures. Also added a pull policy to pull latest images. I think this might be needed for CI runs against staging versions.

Reply:
This would make the build unstable. Half of the future builds don't contain all the plugins, so this would make CI pass/fail intermittently for other people's PRs.